src/constant/movie.ts

```diff
@@ -0,0 +1,30 @@
+export const LIST_STATE = {
+  POPULAR: 'popular',
+  SEARCHED: 'searched',
+};
+
+export const LIST_HEADING = (state: ListState, movieName: string) => {
+  if (state === LIST_STATE.POPULAR) return '지금 인기 있는 영화';
+  if (state === LIST_STATE.SEARCHED)
+    return `"${movieName.replace("'", '＇')}" 검색 결과`;
+};
+
+export const TOGGLE_SKELETON = {
+  SHOW: 'show',
+  HIDDEN: 'hidden',
+} as const;
+
+export const SKELETON_REPEAT_TIME = 20;
+
+export const MAX_MOVIE_QUANTITY_PER_PAGE = 20;
+
+export const BREAKPOINT_SMALL = 410;
+
+export const COMMENT: { [key: number]: string } = {
+  0: '별점은 매겨주세요.',
```

Comment:
Score에 대한 타입을 선언해 두었으니...

```suggestion
export const COMMENT: Record<Score, string> = {
  0: '별점은 매겨주세요.',
```

이렇게 변경하면 어떨까요?